tests/queries/0_stateless/03200_reverse_by_separator.reference

```diff
@@ -0,0 +1,26 @@
+com.google.www\0
```

Comment:
Why do we add \0 at the end, which is not present in the original input? In my opinion, the function should only reorder tokens but keep the string content the same. For example, it's not idempotent if we apply it multiple times:

```SQL
SELECT
    reverseBySeparator('www.google.com' AS c),
    reverseBySeparator(reverseBySeparator(c)),
    c
FORMAT TSV

Query id: c4e98bc7-9c12-4f0e-b7f8-aa0bcda21353

com.google.www\0	www\0.google.com\0	www.google.com
```